kgo.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const kgo = require('kgo');
+const shuv = require('shuv');
+const foreign = require('foreign');
+const readFile = (filename, callback) => fs.readFile(filename, {encoding: 'utf8'}, callback);
+const concat = kgo.sync(strings => strings.join(''));
+const complete = (writeError, writeOut) => (error, result) => {
+  if(error == null){
+    writeOut(result);
+    process.exit(0);
+  }else{
+    writeError(String(error) + '\n');
+    process.exit(1);
+  }
+};
+
+const main = () => {
+  const relative = (filename) => path.join(process.argv[2], filename);
+  const resolveIndex = (index) => index.match(/^.*(?=\n)/gm).map(relative);
```

Comment:
Thanks to #15 we can now use `S.lines(index)` in place of `index.match(/^.*(?=\n)/gm)`.